OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.cs

```diff
@@ -85,6 +85,19 @@ public static IEnumerable<object[]> InvalidMinuteCases()
             };
         }
 
+        private static TeacherStatus GetInValidTeacherStatus()
+        {
+            int maxTeacherStatus = (int)Enum.GetValues(typeof(TeacherStatus))
+                                .Cast<TeacherStatus>().Count();
```

Comment:
Take this whole block here and put it on the next line:

```csharp
            int maxTeacherStatus =
                (int)Enum.GetValues(typeof(TeacherStatus))
                    .Cast<TeacherStatus>().Count();
```

Reply:
It is fixed